feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/FloorMapScreen.kt

```diff
@@ -69,26 +87,71 @@ private fun FloorMapScreen(
     uiState: FloorMapScreenUiState,
     snackbarHostState: SnackbarHostState,
     onSideEventClick: (SideEvent) -> Unit,
+    onClickFloorLevelSwitcher: (FloorLevel) -> Unit,
 ) {
     Scaffold(
-        modifier = Modifier.testTag(FloorMapScreenTestTag),
+        contentWindowInsets = WindowInsets(0.dp), // https://stackoverflow.com/a/75962622
         snackbarHost = { SnackbarHost(snackbarHostState) },
-        content = { padding ->
-            Column(
+        modifier = Modifier
+            .testTag(FloorMapScreenTestTag)
+            .statusBarsPadding(),
+        content = { innerPadding ->
+            Box(
                 Modifier
-                    .padding(padding),
+                    .padding(innerPadding),
             ) {
-                Text(
-                    text = "Please implement FloorMapScreen!!!",
-                    style = MaterialTheme.typography.titleLarge,
-                )
-                FloorMap(floorLevel = uiState.floorLevel)
-                FloorMapSideEventList(
-                    uiState = uiState.floorMapSideEventListUiState,
-                    onSideEventClick = onSideEventClick,
-                    modifier = Modifier.weight(1f),
+                Column(
+                    Modifier
+                        .fillMaxSize(),
+                ) {
+                    Text(
+                        text = "Please implement FloorMapScreen!!!",
+                        style = MaterialTheme.typography.titleLarge,
+                    )
+                    FloorMap(floorLevel = uiState.floorLevel)
+                    FloorMapSideEventList(
+                        uiState = uiState.floorMapSideEventListUiState,
+                        onSideEventClick = onSideEventClick,
+                        modifier = Modifier
+                            .weight(1f)
+                            .fadingEdge(
+                                Brush.verticalGradient(
+                                    0.85f to Color.Black,
```

Comment:
Designer says that she wanted to use the surface Color for the gradient.  (The reason is a technical reason)
![image](https://github.com/DroidKaigi/conference-app-2023/assets/1386930/f3527700-906b-4cec-858e-bec43386a85d)